scripts/compare_signatures.py

```diff
@@ -0,0 +1,98 @@
+import re
```

Comment:
Can you move this script and its folder to `.github/scripts/compare_signatures.py`?